contracts/implementation/AutoClaim.sol

```diff
@@ -209,6 +209,12 @@ contract AutoClaim is IAutoClaim {
         return request.updateIntervalId > 0 && request.updateIntervalId < currentUpdateIntervalId;
     }
 
+    function unchecked_inc(uint256 i) private pure returns (uint256) {
+        unchecked {
+            return i + 1;
```

Comment:
Not a change we need to make but a bit of a curio I suppose. I wonder if this yields *any* gas savings at all?

```suggestion
            return i++;
```

Somewhat doubtful off the top of my head given there's no dedicated increment instruction I don't think?

Reply:
@0xhrithik I meant the increment vs addition.